src/WebJobs.Script/Host/ScriptHostManager.cs

```diff
@@ -76,8 +92,16 @@ public ScriptHostManager(ScriptHostConfiguration config, IScriptEventManager eve
             EventManager = eventManager ?? new ScriptEventManager();
 
             _structuredLogWriter = new StructuredLogWriter(EventManager, config.RootLogPath);
+            _performanceManager = hostPerformanceManager ?? new HostPerformanceManager(settingsManager);
+
+            if (config.HostHealthMonitorEnabled && settingsManager.IsAzureEnvironment)
```

Comment:
>settingsManager.IsAzureEnvironment [](start = 51, length = 34)

Do we need this?

Reply:
I'm just trying to avoid having a noop timer running in cases where we're running locally, etc. The host health check currently relies strictly on an env var that comes from the sandbox, so if not running in the sandbox, it'd all be wasted effort.